fix/link_fixdirs.sh

```diff
@@ -52,13 +52,17 @@ am_ver=${am_ver:-20220805}
 orog_ver=${orog_ver:-20220805}
 sfc_climo_ver=${sfc_climo_ver:-20221017}
 
```

Comment:
@KateFriedman-NOAA Will this change work with the workflow?

Reply:
In terms of clone, build, link it works fine. Did not try running with your branch since we haven't moved forward with UFS_UTILS hash yet. Will put output from clone, build, link test on WCOSS2 into comment in PR.